nilearn/surface/surface.py

```diff
@@ -568,6 +611,42 @@ def load_surf_data(surf_data):
     return data
```

Comment:
maybe we should check for collections.Iterable rather than ndarray? a list should be ok

Reply:
Where you mean ? At this line ```isinstance(surf_data, np.ndarray)``` ?

Like this ```isinstance(surf_data, collections.Iterable)``` ?

Reply:
that's what I had in mind in case a user gives a list but actually you can ignore, I guess forcing ndarray makes sense here.

Reply:
Documentation asks to give nd.array so we can leave this as it is for now.

Reply:
ok, I agree